server/routes/index.ts

```diff
@@ -90,6 +90,11 @@ export default function routes(services: Services): Router {
     courtCaseRoutes.submitTaskList,
   )
 
+  post(
+    '/person/:nomsId/:addOrEditCourtCase/:courtCaseReference/:addOrEditCourtAppearance/:appearanceReference/submit-draft-task-list',
```

Comment:
I'd have hoped for a generic `/submit-draft` so that for every page it would be add the same button with the same submit-draft endpoint called